docs/fides/docs/guides/generate_resources.md

```diff
@@ -65,12 +71,30 @@ dataset:
       data_categories: []
       data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
 ```
-<!-- TODO: Link to the `annotate dataset` usage documentation below, when it exists. -->
-The resulting file still requires annotating the dataset with data categories to represent what is stored. 
+The resulting file still requires annotating the dataset with data categories to represent what is stored.
+
+<!-- TODO: Add a section for `annotate dataset` usage below -->
```

Comment:
do we want to add this in here?

Reply:
I think so, this comment has been moved around a bit so we can determine that when someone is going to work on it